src/main.cpp

```diff
@@ -59,6 +59,14 @@ int main(int argc, char *argv[])
 	// QML-GUI
 	//
 
+	if (strlen(qgetenv("QT_QUICK_CONTROLS_STYLE")) == 0)
```

Comment:
Please use `qgetenv("QT_QUICK_CONTROLS_STYLE").isEmpty()`; AFAIK `strlen()` is a C function and using Qt's own function on a QByteArray is safer.

Alternatively, you could also do `qgetenv("QT_QUICK_CONTROLS_STYLE").trimmed().isEmpty()`, that would first remove all characters like spaces, tabs (at the beginning) and newlines (beginning/end) and then check the size.

Reply:
nice, I still have to learn to just use Qt stuff for everything :)